snuba/admin/static/capacity_management/allocation_policy.tsx

```diff
@@ -135,12 +157,23 @@ function AllocationPolicyConfigs(props: {
             "Type",
             "Actions",
           ]}
-          rowData={configs.map((config) =>
-            getReadonlyRow(config, () => enterEditMode(config))
-          )}
+          rowData={configs
+            .filter((config) => Object.keys(config.params).length > 0)
+            .map((config) =>
+              getReadonlyRow(config, () => enterEditMode(config))
+            )
+            .map((row_data) => [
+              row_data.name,
+              row_data.params,
+              row_data.value,
+              row_data.description,
+              row_data.type,
+              row_data.edit,
+            ])}
           columnWidths={[3, 3, 2, 5, 1, 1]}
-          customStyles={createCustomTableStyles({headerStyle: {backgroundColor: getTableColor(policy.configs)}})}
-
+          customStyles={createCustomTableStyles({
+            headerStyle: { backgroundColor: getTableColor(policy.configs) },
+          })}
```

Comment:
You need this in the global table as well

Reply:
otherwise you're gonna have one table changing colors and the other not

Reply:
Good catch, thank you